netman/api/switch_api.py

```diff
@@ -76,6 +76,7 @@ def hook_to(self, server):
         server.add_url_rule('/switches/<hostname>/interfaces/<path:interface_id>/auto-negotiation', view_func=self.unset_interface_auto_negotiation_state, methods=['DELETE'])
         server.add_url_rule('/switches/<hostname>/interfaces/<path:interface_id>/mtu', view_func=self.set_interface_mtu, methods=['PUT'])
         server.add_url_rule('/switches/<hostname>/interfaces/<path:interface_id>/mtu', view_func=self.unset_interface_mtu, methods=['DELETE'])
+        server.add_url_rule('/switches/<hostname>/interfaces/mac-adresses', view_func=self.get_mac_addresses, methods=['GET'])
```

Comment:
Sorry I made I typo,
it seems to be with two D
https://en.wikipedia.org/wiki/MAC_address